content/jmccann/hipchat/index.md

```diff
@@ -50,12 +50,18 @@ The Hipchat plugin supports reading credentials from the Drone secret store. Thi
 ```diff
 pipeline:
   slack:
-    image: plugins/hipchat
+    image: jmccann/drone-hipchat
     room: my-room
 -   auth_token: my-auth-token
 ```
 
-The above `auth_token` Yaml attribute can be replaced with the `HIPCHAT_AUTH_TOKEN` secret environment variable. Please see the Drone documentation to learn more about secrets.
+The above `auth_token` Yaml attribute can be replaced with the `HIPCHAT_AUTH_TOKEN` secret environment variable.
+Please see the [Drone documentation](http://readme.drone.io/0.5/secrets-with-plugins/) to learn more about secrets.
```

Comment:
Either approach is fine to me.  Let me know what you decide on and I can make the update. :)